web/packages/teleport/src/Discover/Database/EnrollRdsDatabase/EnrollRdsDatabaseEnroll.story.tsx

```diff
@@ -40,6 +40,7 @@ import {
 import { EnrollRdsDatabase } from './EnrollRdsDatabase';
 
 const defaultIsCloud = cfg.isCloud;
+const databasesPathWithoutQuery = cfg.api.databasesPath.split('?')[0];
```

Comment:
> an api path with query params (with a question mark), makes stories break (and it starts breaking other stories). i found this [issue](https://github.com/mswjs/msw/issues/71) which i took it to mean it's not supported (how did it work before)? i worked around this by stripping the affected path anything after the `?`

It might have been something related to msw 1.x, I vaguely remember getting this error in v7 update but I'm not sure.

Currently, I see this when I open `http://localhost:9002/?path=/story/teleporte-accesslists-list--list-unlimited`.

> [MSW] Found a redundant usage of query parameters in the request handler URL for "GET /v1/webapi/roles?startKey=&search=&limit=". Please match against a path instead and access query parameters using "new URL(request.url).searchParams" instead. Learn more: https://mswjs.io/docs/recipes/query-parameters

It comes from [`e/web/teleport/src/AccessListManagement/CreateAccessList/CreateAccessList.story.tsx`](https://github.com/gravitational/teleport.e/blob/ba1271c6faff653308e03292dcaacef918c8a1be/web/teleport/src/AccessListManagement/CreateAccessList/CreateAccessList.story.tsx#L66-L68).

I believe we should figure out why it was working, why it got broken, and how we're going to address this. Manually splitting the path in each story might not be the best way long term.

Reply:
Perhaps it'd be possible to override `cfg.api` within storybook so that it returns some kind of a proxy object with a custom getter that always strips the query params?

Maybe there's a simpler way to replace the whole `cfg.api` in storybook with a version that doesn't use query params. We probably don't want to touch `teleport/src/config.ts` itself that much.

Reply:
I believe we can setup something that will remove query params. i saw it somewhere in the msw docs, I'll look around again today

Reply:
It would be nice if the query params weren't in the cfg, it's one of the things blocking the upgrade to react-router 6 😔

Completely out of scope for this PR though of course